### PR TITLE
document few more features + more HTTPS time documentation

### PR DESCRIPTION
**Pull request by x86pup**

### CODE REVIEW

File: static/features.html [hunk 1]
```diff
@@ -948,6 +948,9 @@ <h3><a href="#other-features">Other features</a></h3>
                         <li>Apps that don't have any storage permission aren't allowed to read the
                         list of all user-created directories (this is allowed on Android). The list of
                         files is hidden from such apps on both Android and GrapheneOS.</li>
+                        <li>Screenshot sound is toggleable using the "Touch sounds" option unlike
+                        standard behaviour where it's controlled by the device being on silent
```
Comment (thestinger) [inline static/features.html]: Need to make this clearer since this currently implies that no longer works.

File: static/faq.html [hunk 2]
```diff
@@ -806,13 +806,17 @@ <h3><a href="#default-connections">What kind of connections do the OS and bundle
...
-                            standard network time update implementation, which uses the cellular network
-                            when available with a fallback to SNTP when it's not available. Network time
-                            updates are security sensitive since certificate validation depends on having
-                            an accurate time, but the standard NTP / SNTP protocols used across most OSes
-                            have no authentication.</p>
+                            <p>An HTTPS connection is made to https://time.grapheneos.org/generate_204 to
+                            update the time from the custom X-Time header field, which has millisecond
+                            precision, or falls back to the Date header if X-Time is not available with less
+                            precision. GrapheneOS also lowers the system clock drift warning from two seconds
+                            to 250 milliseconds, and the time update threshold from two seconds to fifty
```
Comment (thestinger) [inline static/faq.html]: We lower it from 2000ms to 250ms.

File: static/features.html [hunk 3]
```diff
@@ -951,6 +951,8 @@ <h3><a href="#other-features">Other features</a></h3>
                         <li>Screenshot sound is toggleable using the "Touch sounds" option unlike
                         standard behaviour where it's controlled by the device being on silent
                         or vibration mode.</li>
+                        <li>Compressed APEX module support is disabled as it's not useful for GrapheneOS,
```
Comment (thestinger) [inline static/features.html]: Should be moved to the verified boot section.

**Review by thestinger — CHANGES_REQUESTED** [review]
See review comments.

**Comment by x86pup** [comment]
@thestinger Made requested changes

**Pull request merged**